doc/README.md

```diff
@@ -42,7 +42,19 @@ It is a daemon, that:
 2. HTTP serves the manual, injecting a script that triggers reload on changes
 3. opens the manual in the default browser
 
-### Testing redirects
+### Redirects check
+
+In order to improve the reading experience of the manuals, since [PR 353513](https://github.com/NixOS/nixpkgs/pull/353513), documentation anchors are checked in CI so that they always redirect to a relevant section instead of failing. For example, if support for a package is dropped, resulting in its deletion from the package set, the anchor leading to its documentation should redirect to the release notes section where this case is explained.
+
+The checks are done directly by `nixos-render-docs`, by comparing with a collection of identifiers in `redirects.json` files. More information is provided in `nrd`'s [README](../pkgs/by-name/ni/nixos-render-docs/README.md).
+
+#### Managing identifiers inside redirects.json files
+
+In order to manipulate the `redirects.json`, the `redirects` utility is accessible from the shell provided in the source directory.
+
+While more documentation is provided, you can read its [source code](../pkgs/by-name/ni/nixos-render-docs-redirects/src).
```

Comment:
Provided where?

Reply:
Provided in general, the tool doesn't have much docs so this links contributors to the the tool itself meanwhile. I plan to write a README.md next and point to it, in addition to the actual help from the tool.

Maybe addressing more explicitly that documentation is incoming makes it less confusing?

Reply:
Hm I'm generally inclined not to check in promises into version history. If we don't have anything to say yet, let's not say it and just add information later.